src/test/acceptance/features/do-you-live-in-scotland.feature

```diff
@@ -0,0 +1,20 @@
+Feature: Do you live in Scotland?
+  In order to apply for the HTBHF programme
+  As a potential claimant
+  I want to select whether or not I live in Scotland
+
+  Background:
+    Given I have entered my details up to the do you live in scotland page
+
+  Scenario: Yes and No are displayed
+    Then Yes and No options are displayed
```

Comment:
I'm surprised this works as it's only the are you pregnant steps that defined this. I would update this to include the page name and add that to the do you live in scotland steps.

Reply:
It works fine because it simply calls `findElements` on `WebDriver` so the fact that it's on the Are You  Pregnant page doesn't matter. Either way, I'll move it out into common steps.

Reply:
There were a few other steps that needed to be sorted too, but this has now been done.